recipes/poco/all/conanfile.py

```diff
@@ -200,11 +203,16 @@ def _configure_cmake(self):
         # On Windows, Poco needs a message (MC) compiler.
         with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
             self._cmake.configure(build_dir=self._build_subfolder)
+        # Disable fork
+        if not self.options.get_safe("enable_fork", True):
+            self._cmake.definitions["POCO_NO_FORK_EXEC"] = True
         return self._cmake
 
     def build(self):
         self._patch_sources()
         cmake = self._configure_cmake()
+        # Disable parallel builds because c3i (=conan-center's test/build infrastructure) seems to choke here
+        cmake.parallel = False
```

Comment:
I think we can revert this change. The error came from the `test_package`, not from the library build... so, if needed, it should go in the `test_package/conanfile.py` (but please, let's try it first now that we have added the `MSBUILDDISABLENODEREUSE` envvar)